content/community.html

```diff
@@ -30,6 +30,15 @@
     more elaborate or controversial changes, please discuss them on the mailing
     list before sending a pull request.
   </p>
+  <h1>Commercial</h1>
```

Comment:
s/Commercial/Commercial support/ (caps don't matter, as it'll be all-caps anyways)